Exercises-4-OOP/src/inheritanceanimal/Cat.java

```diff
@@ -0,0 +1,29 @@
+package inheritanceanimal;
+
+public class Cat extends Mammal {
+
+	 String greets = "meow";
+
+	public Cat() {
```

Comment:
This constructor is not part of the requirements, should be removed.

Reply:
have to keep String greets = "meow"; 
otherwise, it won't run.

Reply:
if I remove it, the application won't run :(